domains/client/domain-operator/src/tests.rs

```diff
@@ -972,12 +1112,36 @@ async fn test_invalid_state_transition_proof_creation_and_verification(
     // Get a bundle from the txn pool and modify the receipt of the target bundle to an invalid one
     let (slot, bundle) = ferdie.produce_slot_and_wait_for_bundle_submission().await;
     let original_submit_bundle_tx = bundle_to_tx(bundle.clone().unwrap());
+    let original_length;
```

Comment:
Perfer to return `original_length` in the block like `bad_receipt_hash` instead of define with no init value (it is quite OO style), non-blocker.